twilight-model/src/poll/media.rs

```diff
@@ -0,0 +1,71 @@
+use crate::id::{marker::EmojiMarker, Id};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct PollMedia {
+    /// The emoji of the field.
+    ///
+    /// When creating a poll answer with an emoji, one only
+    /// needs to send either the id (custom emoji) or name
+    /// (default emoji) as the only field.
+    pub emoji: Option<PartialPollMediaEmoji>,
+    /// The text of the field.
+    pub text: Option<String>,
```

Comment:
This one is a bit strange, I think it should be skipped if it is left out since that is what the docs say. But the docs also say that it may be nullable in the future, so does it need to be able to be both???

I think I will just say to add:
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub text: Option<String>,
```